dts/bindings/led/issi,is31fl3194.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2024 Arduino SA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  IS31FL3194 3-channel LED driver with programmable pattern sequencing
+
+  The LED_SHELL application can be used for testing.
+
+  The following defines a single RGB LED in the is31fl3194 DT node:
+
+  is31fl3194@53 {
+    compatible = "issi,is31fl3194";
+    reg = <0x53>;
+
+    led_0 {
+      label = "RGB LED";
+      index = <0>;
+      color-mapping =
+        <LED_COLOR_ID_RED>,
+        <LED_COLOR_ID_GREEN>,
+        <LED_COLOR_ID_BLUE>;
+    };
+  };
+
+  The following example defines three single-channel LEDs in the is31fl3194 DT node:
+
+  is31fl3194@53 {
+    compatible = "issi,is31fl3194";
+    reg = <0x53>;
+
+    led_0 {
+      label = "RED LED";
+      index = <0>;
+      color-mapping =
+        <LED_COLOR_ID_RED>;
+    };
+
+    led_1 {
+      label = "GREEN LED";
+      index = <1>;
+      color-mapping =
+        <LED_COLOR_ID_GREEN>;
+    };
+
+    led_2 {
+      label = "BLUE LED";
+      index = <2>;
+      color-mapping =
+        <LED_COLOR_ID_BLUE>;
+    };
+  };
+
+
+compatible: "issi,is31fl3194"
+
+include: ["i2c-device.yaml", "led-controller.yaml"]
+
+child-binding:
+  properties:
+    label:
+      required: true
+
+    index:
+      required: true
+
+    color-mapping:
+      required: true
+
+    current-limit:
+      type: int
+      enum:
+        - 10
+        - 20
+        - 30
+        - 40
+      default: 20
+      description: |
+        The current limit for the LED in mA. If not specified,
+        the hardware default value of 20mA is used.
```

Comment:
a justification is a "why"

https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#rules-for-default-values